.coderabbit.yaml

```
@@ -0,0 +1,17 @@
+language: "ko-KR"  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)